nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -219,6 +219,9 @@
 
 - `networking.networkmanager.firewallBackend` was removed as NixOS is now using iptables-nftables-compat even when using iptables, therefore Networkmanager now uses the nftables backend unconditionally.
 
+- Package `luaPackages.haskell-tools-nvim` has been updated to version 2.
```

Comment:
I dont want to sound rude but I dont think it's high-profile enough to deserve a release entry, most software doesn't get an entry here either.

Reply:
Haha yeah, I wasn't sure if I should add it either.